scripts/globals/abilities/elemental_siphon.lua

```diff
@@ -43,8 +43,7 @@ function onUseAbility(player,target,ability)
         spiritEle = 3
     end;
     
-    local pEquipMods = player:getMod(MOD_ENHANCES_ELEMENTAL_SIPHON);
```

Comment:
Well that throws a wrench into things, I'll need to test that bonus values from equips are reported by getMod and if so remove the skill bonus from the method while keeping in bonus effects.